### PR TITLE
Clean package drop share and republish

### PR DESCRIPTION
**Pull request by troydai**
/cc @anurse @pranavkm @davidfowl

### CODE REVIEW

**Comment by troydai** [comment]
I hope to merge this to release.

File: makefile.shade [hunk 1]
```diff
@@ -32,22 +32,15 @@ use-standard-lifecycle
...
-                try
-                {
-                    Directory.Delete(versionDir, recursive: true);
-                }
-                catch (IOException)
-                {
-                    // Ignore.
-                }
-            }
+          // delete every thing
```
Comment (pranavkm) [inline makefile.shade]: Change the target to `publish`. We can't delete everything prior to knowing this is a coherent build.

Comment (troydai) [inline makefile.shade]: ok

**Comment by anurse** [comment]
Looks good to me but I'm unfamiliar with Coherence so I'd rather someone else gives the squirrel

**Comment by pranavkm** [comment]
:shipit:

**Comment by troydai** [comment]
Ping @pranavkm @anurse I change this PR to make it a dry run. See the commit description.

Deleting operation is risky, I decide to approach this in a safer manner.

**Pull request merged**